src/plugins/console/public/types/plugin_dependencies.ts

```diff
@@ -59,6 +59,12 @@ export interface ConsolePluginStart {
    * this function will open the embedded console on the page if it is currently rendered.
    */
   openEmbeddedConsole?: (content?: string) => void;
+  /**
+   * openEmbeddedConsoleAlternateView is available if the embedded console can be rendered.
+   * Calling this function will open the embedded console to the alternative view. If there is no alternative view registered
+   * this will open the embedded console.
```

Comment:
We could make this function no-op if there is not an alternate view registered. I was torn on that.

Reply:
I'm not too opinionated on this, but I think perhaps going for a noop can lead to better UX as it can prevent confusing behaviour for the user. Rather than opening the console in an unexpected state, we don't do anything which might be a clearer indication that there is no alternative view.